README.md

```diff
@@ -333,7 +336,11 @@ Releasing a new version is quite simple, as the packages are build and distribut
 1. Make sure all dependencies are up to date (**especially Rasa SDK**)
     - For Rasa SDK that means first creating a [new Rasa SDK release](https://github.com/RasaHQ/rasa-sdk#steps-to-release-a-new-version) (make sure the version numbers between the new Rasa and Rasa SDK releases match)
     - Once the tag with the new Rasa SDK release is pushed and the package appears on [pypi](https://pypi.org/project/rasa-sdk/), the dependency in the rasa repository can be resolved (see below).
-2. Switch to the branch you want to cut the release from (`main` in case of a major / minor, the current feature branch for micro releases)
+2. In case of a minor release, create a new branch that corresponds to the new release, e.g. 
+   ```bash
+    git checkout -b 2.4.0
+    ```
+3. Switch to the branch you want to cut the release from (`main` in case of a major, the branch you just created in case of a minor, the current feature branch for micro releases)
```

Comment:
```suggestion
3. Switch to the branch you want to cut the release from (`main` in case of a major, the `<major>.<minor>.x` branch for minors and micros)
```